common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -731,7 +732,8 @@ def test_export_to_xml(self, mock_val_api):
         self.descriptor.download_track = True
         self.descriptor.html5_sources = ['http://www.example.com/source.mp4', 'http://www.example.com/source1.ogg']
         self.descriptor.download_video = True
-        self.descriptor.transcripts = {'ua': 'ukrainian_translation.srt', 'ge': 'german_translation.srt'}
+        self.descriptor.transcripts = OrderedDict(
```

Comment:
We can't gaurantee that everyone is passing in orderded dicts in other parts of the system though so I'm not sure if this will work.

What happens if you pass in a regular dict here?